template/razor_repl_expr.go

```diff
@@ -123,8 +126,8 @@ func expressionParserInternal(repl replacement, match string, skipError, interna
 				return result, nil
 			}
 		}
-		if err != nil {
-			InternalLog.Debug(color.CyanString(fmt.Sprintf("Invalid expression '%s' : %v", expression, err)))
+		if !debugMode && err != nil {
```

Comment:
Why not print this when not in debugMode. Trace seems pretty debugMode to me!

Reply:
Same as above. We do not want to print this in all circumstance.